### ISSUE TITLE
zsh-5.7: 'clint' prompt causes segfault in zsh-5.7

### ISSUE THREAD

**Issue by xelkarin** [termux/termux-packages#3315]
zsh-5.7 segfaults when command prompt appears immediately after loading startup files when the included _clint_ prompt theme is selected in `$HOME/.zshrc`.

Add the following to `$HOME/.zshrc`:
```
autoload -U promptinit
promptinit
prompt clint
```

Expected result:
Command prompt should appear allowing commands to be entered into the shell.

`termux-info`:
```
Updatable packages:
All packages up to date
System information:
Linux localhost 3.10.49-g415000d #1 SMP PREEMPT Tue Feb 5 14:39:18 UTC 2019 armv7l Android
Termux-packages arch:
arm
Android version:
7.1.2
Device manufacturer:
Motorola
Device model:
XT1527
```

**Comment by xeffyr** [comment]
Happens on AArch64/Oreo as well.

**Comment by xelkarin** [comment]
I wasn't able to reproduce this on an amd64 system using gcc-7.3.0.

**Comment by xeffyr** [comment]
> amd64 system using gcc-7.3.0

Termux is a lot different and there possible bugs that don't possible on desktop Linux distributions. So trying to reproduce bug on Ubuntu/Debian/RHEL/Arch/somethingelse is *useless*.

**Comment by fornwall** [comment]
@xelkarin Thanks for reporting! This has been fixed in version `5.7.1` of the zsh package - update to it by running `pkg up`!
